app/lib/link.rb

```diff
@@ -0,0 +1,17 @@
+class Link
+  attr_reader :logic
+
+  def initialize from_mac, to_mac, logic
+    @from_board = Board.find_by mac: from_mac
+    @to_board = Board.find_by mac: to_mac
+    @logic = logic
+  end
+
+  def run
+    if logic == "add" and @to_board.get_methods.has_key?(logic.to_sym)
```

Comment:
Vezi ca `and` si `&&` au precedente diferite in ruby 
http://stackoverflow.com/questions/1426826/difference-between-and-and-in-ruby

In cazul asta nu se aplica, doar ca e bun ca practice sa folosesti `&&`. Un exemplu bun:
```
if a = false or true
```
nu se va evalua ca `if a = (false or true)` asa cum te-ai astepta ci ca `if (a = false) or true`